packages/expo-module-template-local/ios/{%- project.name %}.podspec

```diff
@@ -17,8 +17,7 @@ Pod::Spec.new do |s|
   # Swift/Objective-C compatibility
   s.pod_target_xcconfig = {
     'DEFINES_MODULE' => 'YES',
-    'SWIFT_COMPILATION_MODE' => 'wholemodule'
   }
 
-  s.source_files = "**/*.{h,m,mm,swift,hpp,cpp}"
+  s.source_files = "**/*.{h,m,swift}"
```

Comment:
I think it's ok to include all of these file extensions. People might get confused if they add some Objective-C++ or C++ code to the module and it won't link because the podspec doesn't include them.

Reply:
```suggestion
  s.source_files = "**/*.{h,m,mm,swift,hpp,cpp}"
```

Reply:
You didn't apply this change 😉